index.html

```diff
@@ -180,6 +180,10 @@ <h2>
         in the [=document tree=]. It could be in a [=shadow tree=] of
         the same [=document=].
       </p>
+      <p>
+        Each <a>document</a>, <a>input</a> element, and <a>textarea</a> element has a boolean flag
+        <dfn>has scheduled selectionchange event</dfn>, which is initially false.
```

Comment:
I'd remove "flag" here. It's redundant with boolean and we stopped using flag in favor of boolean elsewhere.